lib/server/db/model/Connection.js

```diff
@@ -0,0 +1,38 @@
+var mongoose = require("mongoose");
+
+/**
+ * Validates that a provided identifier is a positive number
+ *
+ * @private
+ * @function validateUserId
+ * @param {Number} id - A User's unique identifier
+ * @returns {Boolean}
+ */
+function validateUserId(id) {
```

Comment:
These IDs are strings. Needs either to be updated to validate as Strings or just be removed entirely. I think it might be safe to assume that we'll only pass in strings as the ids when creating these models, in which case we can lighten the code-load here. What do you think?